README.md

```diff
@@ -10,6 +10,8 @@
 
 The Accessibility Engine for automated testing of HTML-based user interfaces. Drop the aXe on your accessibility defects!
 
+Thanks for helping us to make axe-core the most widely used HTML accessibility testing library in the Galaxy! Share your axe-core story using #axecoresquad or by submitting it [here](https://accessibility.deque.com/axe-core-stories) to get a sweet T-shirt
```

Comment:
I'd lowercase "Galaxy" and put a period at the end to complete the sentence, but otherwise looks good to me.